src/constants/routes.js

```diff
@@ -4,4 +4,10 @@ export const SIGN_IN = '/signin';
 export const HOME = '/home';
 export const ACCOUNT = '/account';
 export const ADMIN = '/admin';
-export const PASSWORD_FORGET = '/pw-forget';
\ No newline at end of file
+export const PASSWORD_FORGET = '/pw-forget';
```

Comment:
Is this needed/being used somewhere?

Reply:
This was one of the example routes that the skeleton was given to us with, maybe its used by firebase somewhere?

Reply:
Oh I see it was just moved down a line. Ok I can fix that myself and remove, no worries.